content/foundation/governing-board/index.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Governing Board"
+weight = 3
+extra.summary = """
+The Governing Board is an advisory board that is made up of elected representatives from all across the Matrix ecosystem. The role of the Governing Board is to offer guidance and support to the Guardians, Foundation staff, and Spec Core Team.
+"""
++++
+
+The Governing Board is an advisory board that is made up of elected representatives from all across the Matrix ecosystem. The role of the Governing Board is to offer guidance and support to the Guardians, Foundation staff, and Spec Core Team. It is organized under [these bylaws](/media/2024-04-governing-board-terms-of-reference.pdf) (v1.2, last updated April 2024).
+
+There are nine different constituency groups that are allocated seats on the Governing Board, across three categories:
+
+* **Community representatives**
+  * 4 seats for Individual Members – these are people who currently provide financial support to the Foundation.
+  * 3 seats for Ecosystem Members – these are FOSS projects and communities around Matrix.
+  * 2 seats for Associate Members – these are FOSS foundations, academic institutions, and other communities that are aligned with key parts of our mission.
+* **Funder representatives**
+  * 4 seats for Platinum Members – these organisations sponsor at the highest level.
+  * 3 seats for Gold Members — these organisations sponsor at a high level.
+  * 2 seats for Silver Members — these organisations sponsor at a level commensurate with the size of their organisation.
+* **Foundation representatives**
+  * 3 seats for Guardians – these are the members of the Foundation's board of directors.
+  * 2 seats for the Spec Core Team – these are the members of the Spec Core Team that looks after the spec.
+  * Managing Director – this is the Foundation's head of staff.
+
+Elections are being held in the second quarter of each calendar year, alternating halves of the Constituency Groups representatives on the Governing Board as per the following calendar:
+
+* April-June 2024, and any subsequent even-numbered years: Guardians, SCT, Platinum and Ecosystem
+* April-June 2025, and any subsequent odd-numbered years: Gold, Silver, Individual, Associate
+
+## Elected representatives
```

Comment:
we might want to point out the board's chairs. i assume there will eventually be info on what that means.

Reply:
Yes, current chair and vice chair is needed. I can send a followup for some info on that.

Reply:
>  I can send a followup for some info on that.

would be much appreciated :+1:

Reply:
Actually I referenced https://matrix.org/blog/2025/01/10/this-week-in-matrix-2025-01-10/#governing-board-website now

Reply:
You missed me & Kim as overall (vice)chair :grin:

Reply:
oooops :D